From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 26 Nov 2017 17:25:08 +0100
Subject: [PATCH] baremetal: Add Armband Openstack repos to kvm, cmp

Armband provides one APT repository that should be added to all
baremetal nodes explicitly (the other Armband repos are now part of
`linux.system.repo.mcp.openstack` or `linux.system.repo.mcp.extra`):
- linux.system.repo.mcp.armband;

This repo is pre-installed for all infrastructure VMs, so we only
handle it via Salt for baremetal kvm* and cmp* nodes.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml     | 1 +
 .../classes/cluster/mcp-pike-common-ha/openstack_compute.yml     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml b/mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml
index df230452..5bbe0fe4 100644
--- a/mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml
+++ b/mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml
@@ -7,6 +7,7 @@
 ##############################################################################
 ---
 classes:
+  - system.linux.system.repo.mcp.armband
   - system.linux.system.repo.glusterfs
   - service.keepalived.cluster.single
   - system.glusterfs.server.volume.glance
diff --git a/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml b/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml
index 24acbacf..e9f95f2d 100644
--- a/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml
+++ b/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml
@@ -8,6 +8,7 @@
 ---
 classes:
   - system.linux.system.repo.mcp.openstack
+  - system.linux.system.repo.mcp.armband
   - system.linux.system.repo.glusterfs
   - system.glusterfs.client.cluster
   - system.nova.compute.cluster
